Animated story video generation

Yousif-GO opened the pull request:
This Code demonstrates how to generate an animated story video by:

1. Generating a story sequence using structured Google Gemini API (for character consistency).
2. Generating images for each scene using Google’s Imagen API.
3. Synthesizing narration audio using Kokoro's KPipeline.
4. Creating short video clips (image + audio overlay) for each scene.
5. Combining all clips into one final video.
6. Cleaning up temporary files after processing.

https://github.com/user-attachments/assets/4df87465-7c81-482b-8193-b21803ff3510

markmcd commented:
This is very cool!  I love that it's a complex end-to-end example, contains text instructions throughout, and uses Google models and open-source tools exclusively. Plus it's fun!

I'll go over in a bit more detail when I get some time but for now, would you be able to use the [`google-genai` SDK](https://pypi.org/project/google-genai) instead of `google-generativeai`? It'd be great to show this example off, and having the latest SDK makes it a useful demo.

Have you tried doing the audio generation using Gemini 2.0 by any chance? It's only available through the [Live API](https://ai.google.dev/gemini-api/docs/multimodal-live) now. But there is also [a preview](https://www.youtube.com/watch?v=qE673AY-WEI) that we can look at getting you onto.  This isn't required at all, but something we'll want to do once audio generation becomes GA.

And thank you for following our contrib template! Do you want to add your name on this anywhere? You're welcome to add a byline at the top with a link to a social account or website.

Yousif-GO commented:
Hi Mark,

Thank you for your feedback. I just created another [pull request](https://github.com/google-gemini/cookbook/pull/456) with the suggested changes. The new code now provides a full demonstration of Gemini text, Live API, and Imagen working together to generate a story video.

Notable changes include:
- Removed the use of Kokoro and added Gemini Live instead.
- Tweaked various parts of the code to ensure smooth functionality.
- Updated to the latest [google-genai SDK](https://pypi.org/project/google-genai) rather than using google-generativeai.

Also ,  an access to Native audio output would also be great for additional experimentation.

I'm glad you liked it, and I hope this demo could effectively showcases the capabilities of Gemini and Imagen in a fun and engaging way.
